koswat/strategies/strategy_location_reinforcement.py

```diff
@@ -40,3 +40,22 @@ def get_reinforcement_costs(
                 f"Reinforcement {reinforcement_type.output_name} not available, costs cannot be computed.",
             )
         return self.strategy_location_input.get_reinforcement_costs(reinforcement_type)
+
+    def get_infrastructure_costs(
+        self, reinforcement_type: type[ReinforcementProfileProtocol]
+    ) -> tuple[float, float]:
+        """
+        Returns the infrastructure costs for the given reinforcement type (without and with surtax).
+
+        Args:
+            reinforcement_type (type[ReinforcementProfileProtocol]): Reinforcement type.
+
+        Returns:
+            tuple[float, float]: Tuple containing the infrastructure costs without and with surtax.
+        """
+        if (
+            not self.strategy_location_input
+            or reinforcement_type not in self.available_measures
+        ):
+            return (0.0, 0.0)
```

Comment:
inverse for better readability:
`if self.strategy_location_input and reinforcement_type in self.available_measures`

Reply:
Fair point